src/components/CircleGraph/CircleGraph.js

```diff
@@ -1,6 +1,7 @@
 import React, { useCallback } from 'react'
 import PropTypes from 'prop-types'
-import { Spring, animated } from 'react-spring/renderprops'
+import { Spring, animated } from '@react-spring/web'
+// import { Spring, animated } from 'react-spring/renderprops'
```

Comment:
Maybe clean it up?

Reply:
Have some reason to keep the line 4 commented instead remove? Just curious @reznord